docs/nidcpower/rep_caps.rst

```diff
@@ -18,10 +18,15 @@ Repeated Capabilities
 
     Repeated capabilities attributes use the indexing operator :python:`[]` to indicate the repeated capabilities.
     The parameter can be a string, list, tuple, or slice (range). Each element of those can be a string or
-    an integer. If it is a string, you can indicate a range using the same format as the driver: :python:`'0-2'` or
-    :python:`'0:2'`
+    an integer.
 
-    Some repeated capabilities use a prefix before the number and this is optional
+    ..
+        If it is a string, you can indicate a range using the same format as the driver: :python:`'0-2'` or
+        :python:`'0:2'`
```

Comment:
I would remove this. We are discouraging Python users from using "0-2" and instead should use [0,1,2] or some more pythonic thing

Reply:
Github does a terrible job of highlighting what we want to talk about.
Are you referring to the

"If it is a string, you can indicate a range using the same format as the driver: :python:`'0-2'` or :python:`'0:2'`"? That's a comment that won't show up except in the raw .rst, by the way.

Reply:
I see better in the conversation that is what you're referring to.
Yeah, I can see how that wouldn't even be useful for contributors. It's not an implementation detail, but rather a tip for how to do something that we don't want to do in the first place. I'll remove it.